middlewares/express.js

```diff
@@ -0,0 +1,25 @@
+import { loader } from '../lib/middlewares/loader.js'
+
+/**
+ * Import a plain Express middleware.
+ *
+ * Configuration fields:
+ *  - module (string, required): the name of the NPM module to load
+ *  - options (any, optional): some options to pass to the Express middleware
+ *
+ * @param {*} trifid Trifid object containing the configuration, and other utility functions.
+ * @returns Express middleware.
+ */
+const factory = async (trifid) => {
+  const { config } = trifid
+  const { module, options } = config
+  if (!module) {
+    throw new Error("configuration is missing 'module' field")
```

Comment:
I think  'requires' is a better word for these Parameter error

configuration requires 'module' field

Reply:
Or perhaps, if one want's to be more verbose (and developer-friendly)

configuration requires 'module' field,  specifying the middleware's NPM module

Reply:
This makes sense, I will update it.
Thanks for the suggestion!